src/EPR.Accreditation.Facade/Controllers/SiteController.cs

```diff
@@ -0,0 +1,27 @@
+﻿using EPR.Accreditation.Facade.Services.Interfaces;
+using Microsoft.AspNetCore.Mvc;
+
+namespace EPR.Accreditation.Facade.Controllers
+{
+    [ApiController]
+    [Route("/api/[controller]")]
+    public class SiteController : ControllerBase
+    {
+        protected readonly ISiteService _siteService;
+        
+        public SiteController(
+            ISiteService siteService)
+        {
+            _siteService = siteService ?? throw new ArgumentNullException(nameof(siteService));
+        }
+
+        [HttpGet("Site/{siteExternalId}")]
+        public async Task<IActionResult> GetSite(
+            Guid siteExternalId)
+        {
+            var site = await _siteService.GetSite(siteExternalId);
```

Comment:
You should get ALL your information within Facade (basing this comment off of what I have said in Portal). Do you calls and put the information together here if that is necessary

Reply:
Done.